examples/suspense_tests/src/app.rs

```diff
@@ -135,6 +135,12 @@ fn SecondaryNav() -> impl IntoView {
     }
 }
 
+/*
```

Comment:
We should set this up either as a component documentation or as a module documentation perhaps

Reply:
I'm not sure about this honestly. The distinction I am trying to make with this comment is implementation specific and not specific to the module/component. I could be convinced otherwise, but that's how I feel at the moment.

Reply:
I'm not sure what the comment is supposed to mean, to be honest: 

> However, you should use the provided Resource::map method to avoid any potential pitfalls.

What are the potential pitfalls? `.map()` is really just syntax sugar.

I'd suggest either clarifying or removing, so that users don't ask "Wait, what are the pitfalls of `.get().map()`?